### PR TITLE
Better noise texture for 1D and 2D (and an ok option for 3D)

### PR DESCRIPTION
**Pull request by bsavery**
EFFECT:  Better support for 1D and 2D noise.  

Notes: 
Does not support 3D + 4D noise.  These never should have worked.  Perhaps something could be done to combine multiple noise calls in XY, YZ, ZX dimensions to make 3D?

### CODE REVIEW

**Comment by bsavery** [comment]
untitled

File: src/rprblender/nodes/blender_nodes.py [hunk 1]
```diff
@@ -1631,27 +1631,136 @@ def export_hybrid(self):
...
+# this is a matrix of offsets prefilled
+# 3 with X offset, 4 with XY offset, 5 with XYZ offset etc.
+# each list of offsets is seeded by the n in length of offsets
+
+RANDOM_OFFSET = []
+for num in range(4):
+    offsets = []
+    for seed in range(num + 3):
+        offset = [0.0, 0.0, 0.0, 0.0]
+        random.seed(seed)
```
Comment (ardraeiss) [inline src/rprblender/nodes/blender_nodes.py:1644]: Missed "random" module import.

**Comment by ardraeiss** [comment]
Performance is also affected by the number of used arithmetic nodes.

**Comment by bsavery** [comment]
Closing PR.  I am unhappy with performance in RPR 1.0.  RPR 2.0 will have a better solution next week.

**Pull request closed without merging**